pytest_unordered.py

```diff
@@ -48,6 +54,11 @@ def pytest_assertrepr_compare(config, op, left, right):
         left_repr = saferepr(left)
         right_repr = saferepr(right)
         result = ["{} {} {}".format(left_repr, op, right_repr)]
+        left_type = left._expected_type if isinstance(left, UnorderedList) else type(left)
+        right_type = right._expected_type if isinstance(right, UnorderedList) else type(right)
+        if left_type and right_type and left_type != right_type:
+            result.append("Type mismatch:")
+            result.append("{l} != {r}".format(l=left_type, r=right_type))
```

Comment:
"l" looks similar to "I" and "1". Please use another name.

Reply:
good point